scripts/linux/buildImage.sh

```diff
@@ -20,7 +20,7 @@ PROJECT=
 DOCKERFILE=
 DOCKER_IMAGENAME=
 DEFAULT_DOCKER_NAMESPACE="microsoft"
-DOCKER_NAMESPACE=$DEFAULT_DOCKER_NAMESPACE
+DOCKER_NAMESPACE=${DEFAULT_DOCKER_NAMESPACE}
```

Comment:
did you run shellcheck?

Reply:
Not really, I used IntelliJ IDEA while working on this PR. I just fixed all suggestions IDE told me. Usually, suggestions are worth to fix. Do you think I need to double check with another tool?

Reply:
it is great to run shellcheck if not take too long time to do it.

Reply:
It's definitely a great tool. Just pushed fixes for warnings it generated